kratos/processes/simple_mortar_mapper_process.cpp

```diff
@@ -918,6 +988,8 @@ Parameters SimpleMortarMapperProcess<TDim, TNumNodes, TVarType, TNumNodesMaster>
     {
         "echo_level"                       : 0,
         "using_average_nodal_normal"       : true,
+        "compute_discontinuous_interface"  : false,
+        "discontinous_interface_factor"    : 1.0e-4,
```

Comment:
Just for my understanding, what is this factor for?

Reply:
It weighs the discontinuity gap, so in case the gap between not watered conditions will reduce the influence of the conditions in very big gaps

Reply:
So in case the discontinuity between neighbouring entities is large, the larger is this parameter the less the neighbouring entities are taken into account. Am I wrong?

Reply:
Indeed, that's why I haven't hardcoded the value, so in case the discontinuities are bigger than expected you can manually change that

Reply:
Just for me to completely understand:
- Is the parameter bounded between 0 (take everything into consideration) and 1 (do not consider the discontinuous neighbours)?
- Why don't we set it to zero by default? (of course this assumes that the discontinuities exist but are somehow negligible)

Reply:
It is not between 0 and 1, it is between 0 and infinity. 0 is as you said, one means that the nodes in a distance of 1 length of the geometry will be taken into account completely, and you can consider even greater values, but the results you get can be whatever

Reply:
I prefer to consider continuous mapping by default